package.json

```diff
@@ -31,5 +31,5 @@
     "pretest": "node-gyp rebuild -C test",
     "test": "node test"
   },
-  "version": "0.3.1"
+  "version": "0.3.2"
```

Comment:
My understanding is that the usual practice for `0.x.y` versions is the `x` indicates breaking changes and the `y` non-breaking changes. [That is what `node-semver` assumes.](https://github.com/npm/node-semver#caret-ranges-123-025-004)